### PR TITLE
Warn about encryption / character issue

### PR DESCRIPTION
**Pull request by plaindocs**
Fix for #1059 on https://github.com/travis-ci/docs-travis-ci-com/compare/sw-pypi?expand=1#diff-f522233148651d3b9422d7c90d555ac3R27

And some minor text and heading tweaks while I was there.

@iriberri @joepvd take a look if you fancy.

### CODE REVIEW

File: user/deployment/pypi.md [hunk 1]
```diff
@@ -15,16 +17,14 @@ deploy:
...
+[encrypt](/user/encryption-keys/) your password using the Travis CI command line client:
 
 ```bash
 travis encrypt --add deploy.password
 ```
 
-This prompts you to enter the password, and prints an encrypted version
-of it which you should put in your ``.travis.yml`` file as described
-under [Encrypting Sensitive Data](/user/encryption-keys/):
+> Note that if your PiPI password contains [special characters](/user/encryption-keys#Note-on-escaping-certain-symbols) you need to escape them before encrypting your password. Some people have [reported difficulties](https://github.com/travis-ci/dpl/issues/377) connecting to PyPI with passwords containing anything except alphanumeric characters.
```
Comment (iriberri) [inline user/deployment/pypi.md]: s/PiPI/PyPI

File: user/deployment/pypi.md [hunk 2]
```diff
@@ -84,19 +78,19 @@ By default, Travis CI will only release from the **master** branch.
 
 Builds triggered from Pull Requests will never trigger a release.
 
-### Releasing to a different index:
+## Releasing to a self hosted pypi
 
-If you wish to release to a different index you can do so:
+To release to a different pypi index:
```
Comment (iriberri) [inline user/deployment/pypi.md]: "PyPI" instead for consistency?

**Comment by iriberri** [comment]
These are some great updates, @plaindocs! ✨

File: user/deployment/pypi.md (hunk 2 above)
Comment (plaindocs) [inline user/deployment/pypi.md]: OMG, I must have been asleep, or just having an inconsistent day.

**Pull request merged**